src/platform/android.rs

```diff
@@ -62,7 +62,7 @@
 //! If your application is currently based on `NativeActivity` via the `ndk-glue` crate and building
 //! with `cargo apk`, then the minimal changes would be:
 //! 1. Remove `ndk-glue` from your `Cargo.toml`
-//! 2. Enable the `"android-native-activity"` feature for Winit: `winit = { version = "0.30.7",
+//! 2. Enable the `"android-native-activity"` feature for Winit: `winit = { version = "0.30.8",
```

Comment:
We should probably remove the `ndk-glue` -> `android-activity` migration guide at some point, and merely keep the base documentation.